dist/templates/k8s.ovn.org_egressfirewalls.yaml.j2

```diff
@@ -0,0 +1,122 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: (unknown)
+  creationTimestamp: null
+  name: egressfirewalls.k8s.ovn.org
+spec:
+  additionalPrinterColumns:
+  - JSONPath: .status.egressFirewallStatus
+    name: EgressFirewall Status
+    type: string
+  group: k8s.ovn.org
+  names:
+    kind: EgressFirewall
+    listKind: EgressFirewallList
+    plural: egressfirewalls
+    singular: egressfirewall
+  scope: Namespaced
+  subresources: {}
+  validation:
+    openAPIV3Schema:
+      description: EgressFirewall describes the current egress firewall for a Namespace.
+        Traffic from a pod to an IP address outside the cluster will be checked against
+        each EgressFirewallRule in the pod's namespace's EgressFirewall, in order.
+        If no rule matches (or no EgressFirewall is present) then the traffic will
+        be allowed by default.
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+          type: string
+        metadata: 
+          type: object
+          properties:
+            name:
+              type: string
+              pattern: ^default$
+        spec:
+          description: Specification of the desired behavior of EgressFirewall.
+          properties:
+            egress:
```

Comment:
should we call this `egressRules` or simply `rules`?

Reply:
this is what it is called in the openshift version of egressfirewall should I change it here? I was tasked with making it as similar as possible